test/functional/http/post/0.tx.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('./shared');
+var constants = require('../../../../helpers/constants');
+
+var sendTransaction = require('../../../common/complexTransactions').sendTransaction;
+
+describe('POST /api/transactions (type 0)', function () {
+
+	var badTransactions = [];
+	var goodTransactions = [];
+
+	describe('schema', function () {
+
+		var transaction;
+
+		beforeEach(function () {
+			transaction = node.randomTx();
+		});
+
+		describe('sending transactions without proper format', function () {
```

Comment:
I don't think we need this extra context.

This would be fine:

```
POST /api/transactions (type 0)
  schema checks
     using null should fail
 ```